src/horus/engine/driver/board.py

```diff
@@ -103,6 +103,7 @@ def connect(self):
                     # Set current position as origin
                     self.motor_speed(profile.settings['motor_speed_control'])
                     self.motor_acceleration(profile.settings['motor_acceleration_control'])
+                    self.motor_invert(profile.settings['invert_motor'])
```

Comment:
Before this change, motor inversion state would be lost upon connection; this re-applies the motor inversion setting after connecting.